doc/releases/release-notes-3.6.rst

```diff
@@ -546,17 +546,50 @@ Networking
 
 * CoAP:
 
-  * Emit observer/service network events using the Network Event subsystem.
-
+  * Added support for Echo and Request-Tag CoAP options (RFC 9175).
+  * Changed :c:func:`coap_remove_observer` API function return type to bool.
+  * Introduced CoAP service library, which simplifies implementation of CoAP
+    server functionality.
+  * Updated CoAP server example to use CoAP service library.
+  * Added shell module for CoAP server.
+  * Fixed NULL pointer dereference in :c:func:`coap_packet_remove_option`
+  * Added CoAP observer/service network events using the Network Event subsystem.
+  * Changed :c:func:`coap_pending_init` API function to take
+    :c:struct:`coap_transmission_parameters` instead of retry count.
   * Added new API functions:
 
     * :c:func:`coap_get_transmission_parameters`
     * :c:func:`coap_set_transmission_parameters`
+    * :c:func:`coap_handle_request_len`
+    * :c:func:`coap_well_known_core_get_len`
+    * :c:func:`coap_uri_path_match`
+    * :c:func:`coap_packet_is_request`
+    * :c:func:`coap_find_observer`
+    * :c:func:`coap_find_observer_by_token`
+    * :c:func:`coap_pendings_count`
+    * :c:func:`coap_header_set_code`
 
 * Connection Manager:
 
+  * Added a generic Wi-Fi connectivity backend.
+
 * DHCP:
 
+  * Added missing DHCPv6 state structure initialization when initializing
+    network interface.
+  * DHCP-assigned IPv4 address is now removed when interface goes down.
+  * Added DHCPv4 server implementation.
+  * Rearranged DHCP file structure, all DHCP related files are now grouped
+    within ``subsys/net/lib``.
```

Comment:
Should we have here the actual directory so `subsys/net/lib/dhcp`

Reply:
Ok, I've updated to a specific DHCPv4 directory, but since it's different for DHCPv4/6, I've split that bullet into two separate ones, please recheck.